### ISSUE TITLE
Updated RG articles in mediawiki not automatically indexed into production site

### ISSUE THREAD

**Issue by GeoffFroh** [denshoproject/encyc-core#29]
Articles for the Resource Guide that are updated in the backend editors' mediawiki site should be pushed into the production ES index by a scheduled cron job. Currently, this does not appear to be the case. See, for example:

https://resourceguide.densho.org/api/3.0/articles/Farewell%20to%20Manzanar%20Educational%20Kit%20(curricula)/

The api shows the update timestamp as: `"modified": "2018-03-01T22:33:13"`; however, the article in the editors' mediawiki was updated 27 February 2020. (another example: https://resourceguide.densho.org/api/3.0/articles/Hotel%20on%20the%20Corner%20of%20Bitter%20and%20Sweet%20(curricula)/)

It does not appear that this is the case with articles for publication in the regular encyclopedia site as well, but further research is required.

**Comment by gjost** [comment]
Cron jobs were disabled last week (4/29).  I re-enabled them yesterday(? the day before?).

I also ran `encyc publish --force` which re-publishes all articles regardless of whether they have been updated (normally `encyc publish` only updates articles with a newer `modified` datetime than their ES counterparts).

Let me know if things are still not getting published.  If so we may have some faulty logic in there.
